pkg/sentry/arch/arch_aarch64.go

```diff
@@ -145,6 +149,11 @@ func (s *State) StateData() *State {
 	return s
 }
 
+// TLSPointer returns the TPValue address.
+func (s *State) TLSPointer() *uint64 {
```

Comment:
Why does this return a pointer to the field instead of just the value itself?

I think this makes things harder to follow than necessary. I think making ptrace getTLS return the value and then calling ac.StateData().SetTLS(tls) would be more obvious.

Reply:
To return a pointer is just want to store the tls value to the StateData directly. Or we will need to do it in two steps:
1. call t.getTLS(&tls)
2. call ac.setTLS(tls)
which are duplicated operations on x86(fs_base has been restored in the t.getRegs()).

I have updated the commit to use two steps method for reference.